experiments/swdt/Makefile

```diff
@@ -0,0 +1,63 @@
+
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(shell go env GOPATH)/bin
+else
+GOBIN=$(shell go env GOBIN)
+endif
```

Comment:
Shall I write Magefiles to make it portable? :)
This is one of the reasons that I have simple proxies to `go run` the app
- https://github.com/knabben/sig-windows-dev-tools/pull/1

Reply:
it could be a good idea..

my concern is the API machinery tooling to manage controllers and resources portability, there are a lot of boilerplate in kubebuilder tools. 

  I do prefer to build it on Linux with GOOS=windows and publish the .exe at each release to simplify these build scripts (which are complex enough imo)